src/views/home/components/token-list/token-list.view.tsx

```diff
@@ -1,287 +1,274 @@
-                      !isImportedCustomToken &&
-                      customToken.status === "successful" &&
-                      customToken.data.address === token.address;
-                    return (
-                      <div key={token.address} className={classes.tokenButtonWrapper}>
-                        <button
-                          role="button"
-                          className={classes.tokenButton}
-                          onClick={() => onSelectToken(token)}
-                        >
-                          <div className={classes.tokenInfo}>
-                            <Icon url={token.logoURI} size={24} className={classes.tokenIcon} />
-                            <Typography type="body1">{token.name}</Typography>
-                          </div>
-                        </button>
-
-                        {isNonImportedCustomToken ? (
-                          <button
-                            className={classes.addTokenButton}
-                            onClick={() => {
-                              addWrappedToken({ token })
-                                .then((newToken) => {
-                                  setScreen({
-                                    type: "add-token",
-                                    token: newToken,
-                                  });
-                                })
-                                .catch(() => {
-                                  setScreen({
-                                    type: "add-token",
-                                    token,
-                                  });
-                                });
-                            }}
-                          >
-                            <Typography type="body1">Add token</Typography>
-                          </button>
-                        ) : (
-                          <div className={classes.tokenRightElements}>
-                            <Typography type="body2" className={classes.tokenBalance}>
-                              {`${formatTokenAmount(token.balance || BigNumber.from(0), token)} ${
-                                token.symbol
-                              }`}
-                            </Typography>
-                            <button
-                              className={classes.tokenInfoButton}
-                              onClick={() => {
-                                setScreen({
-                                  type: "token-details",
-                                  token: token,
-                                });
-                              }}
-                            >
-                              <InfoIcon className={classes.tokenInfoButtonIcon} />
-                            </button>
-                          </div>
-                        )}
+                  <button
+                    className={classes.addTokenButton}
+                    onClick={() => {
+                      addWrappedToken({ token })
```

Comment:
We could indeed remove this `addWrappedToken` since now [`getTokenFromAddress` already adds the wrapped token](https://github.com/0xPolygonHermez/zkevm-bridge-ui/blob/develop/src/contexts/tokens.context.tsx#L234).

Reply:
Right, done!